### PR TITLE
[SRVOCF-513] Add primitve types returning to Node.js function docs

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
`serverless-docs-1.33`+

Issue:
https://issues.redhat.com/browse/SRVOCF-513

Link to docs preview:
https://77506--ocpdocs-pr.netlify.app/openshift-serverless/latest/functions/reference/serverless-developing-nodejs-functions.html#serverless-nodejs-function-return-values_serverless-developing-nodejs-functions

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/serverless-nodejs-function-return-values.adoc [hunk 1]
```diff
@@ -22,6 +22,98 @@ function handle(context, customer) {
...
+  let somenumber = 100
+  return { body: somenumber }
+}
+----
+
+Calling this function returns the number:
+
+[source,terminal]
+----
+$ curl localhost:8080
```
Comment (jrangelramos) [inline modules/serverless-nodejs-function-return-values.adoc:63]: I think here you can use the same url as the previous one 
```suggestion
$ curl https://myfunction.example.com
```

File: modules/serverless-nodejs-function-return-values.adoc [hunk 2]
```diff
@@ -22,6 +22,98 @@ function handle(context, customer) {
...
+  let someboolean = false
+  return { body: someboolean }
+}
+----
+
+Calling this function returns:
+
+[source,terminal]
+----
+$ curl localhost:8080
```
Comment (jrangelramos) [inline modules/serverless-nodejs-function-return-values.adoc:84]: same here about the url
```suggestion
$ curl https://myfunction.example.com
```

File: modules/serverless-nodejs-function-return-values.adoc [hunk 3]
```diff
@@ -22,6 +22,98 @@ function handle(context, customer) {
...
+----
+
+[source,terminal]
+----
+100
+----
+
+. Example function returning a boolean
```
Comment (jrangelramos) [inline modules/serverless-nodejs-function-return-values.adoc:71]: Here it seems the numbering is not working on the rendered preview. It appears as  "1." (just like the previous one) but I believe it should be rendered as "2."

File: modules/serverless-nodejs-function-return-values.adoc [hunk 4]
```diff
@@ -22,6 +22,98 @@ function handle(context, customer) {
...
+. Example function returning a number
+[source,javascript]
+----
+function handle(context) {
+  let somenumber = 100
+  return { body: somenumber }
+}
+----
+
+Calling this function returns the number:
```
Comment (skrthomas) [inline modules/serverless-nodejs-function-return-values.adoc:59]:
```suggestion
Calling this function returns the following number:
```

File: modules/serverless-nodejs-function-return-values.adoc [hunk 5]
```diff
@@ -22,6 +22,98 @@ function handle(context, customer) {
...
+
+.Example function returning a string
+[source,javascript]
+----
+function handle(context) {
+  return "This function Works!"
+}
+----
+
+Calling this function returns the string:
```
Comment (skrthomas) [inline modules/serverless-nodejs-function-return-values.adoc:38]:
```suggestion
Calling this function returns the following string:
```

File: modules/serverless-nodejs-function-return-values.adoc [hunk 6]
```diff
@@ -22,6 +22,98 @@ function handle(context, customer) {
...
+. Example function returning a boolean
+[source,javascript]
+----
+function handle(context) {
+  let someboolean = false
+  return { body: someboolean }
+}
+----
+
+Calling this function returns:
```
Comment (skrthomas) [inline modules/serverless-nodejs-function-return-values.adoc:80]:
```suggestion
Calling this function returns the following boolean:
```

File: modules/serverless-nodejs-function-return-values.adoc [hunk 7]
```diff
@@ -22,6 +22,98 @@ function handle(context, customer) {
...
+.Example function returning a primitive directly
+[source,javascript]
+----
+function handle(context) {
+  let someboolean = false
+  return someboolean
+}
+----
+
+Calling this function returns:
```
Comment (skrthomas) [inline modules/serverless-nodejs-function-return-values.adoc:103]:
```suggestion
Calling this function returns the following:
```

**Review by skrthomas — COMMENTED** [review]
A few minor comments of the same type for intro sentences. Otherwise, looks good to me.

File: modules/serverless-nodejs-function-return-values.adoc (hunk 3 above)
Comment (rh-max) [reply modules/serverless-nodejs-function-return-values.adoc]: Thanks, fixed!

**Review by skopacz1 — APPROVED** [review]
Everything looks good to me, but please squash your commits and then I can merge this PR

**Comment by rh-max** [comment]
> Everything looks good to me, but please squash your commits and then I can merge this PR

Thanks, done!

**Pull request merged**